### PR TITLE
implement proper data mocking and loading

### PR DESCRIPTION
**Pull request by bonno42h**

### CODE REVIEW

File: src/api/mockData.js
Comment (ddave592) [inline src/api/mockData.js:122]: is there a reason we have `export function getRestaurantData() {` at the top and `export default getRestaurantData;` at the end?

File: src/api/data.js
Comment (ddave592) [inline src/api/data.js:11]: Not a real issue with this just don't understand if this bit of code is being used somewhere or if it's just axios setup for when we have a URL to request data from?

File: src/App.js
Comment (ddave592) [inline src/App.js:38]: I want to add a generic spinner here before merge happens so we can get that done, let me know if you have any objections or if you'd like me to wait until this is merged.

**Review by ddave592 — COMMENTED** [review]
Looking good, thanks for this!

**EDIT:** sorry forgot to add.. I reckon we should delete the data folder as it looks like it might be obsolete after your changes

File: src/api/data.js
Comment (bonno42h) [reply src/api/data.js:11]: It's not being used anywhere just yet. It's mostly a template now.

File: src/api/mockData.js
Comment (bonno42h) [reply src/api/mockData.js:122]: No reason really. I'm used to having an export on a fn and a default export for it at the end just to avoid those errors. 
Not sure how good of a practice is it.

**Review by ddave592 — APPROVED** [review]

**Pull request merged**
